Web Quiz Engine (Kotlin)/task/src/engine/entity/Quiz.kt

```diff
@@ -3,16 +3,24 @@ package engine.entity
 import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonProperty
 import engine.QuizIdentifier
+import jakarta.persistence.*
 import jakarta.validation.constraints.NotBlank
 import jakarta.validation.constraints.Size
 
+@Entity
 data class Quiz(
+    @Id
+    @GeneratedValue
     val id: QuizIdentifier? = null,
     @field:NotBlank(message = "The quiz must have a title")
     val title: String,
     @field:NotBlank(message = "The quiz must have a text")
     val text: String,
     @field:Size(min = 2, message = "The quiz must have at least 2 options")
+    @ElementCollection(fetch = FetchType.EAGER)
     val options: List<String>,
-    @JsonInclude(JsonInclude.Include.NON_NULL) @JsonProperty("answer") val answerList: List<Int>? = null,
+    @ElementCollection(fetch = FetchType.EAGER)
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    @JsonProperty("answer")
+    val answerList: List<Int>? = null,
```

Comment:
You should probably use the same naming that you use in your `AnswerQuizRequestBody`.

Using the same namings as much as possible for these entites will help you once you start using reflective dependent tools like `ModelMapper` or `Mapstruct` in the future.

Reply:
I fully agree. There was a technical limitation. It is fixed now. Done
https://github.com/f8full/hyperskill-web-quiz-engine/pull/2#discussion_r1564736422

Reply:
Also, I ilke those ominous sounding names, can't wait!